includes/class-wc-subscriptions-email.php

```diff
@@ -11,6 +11,24 @@
  */
 class WC_Subscriptions_Email {
 
+	/**
+	 * List of all core subscription email classes.
+	 *
+	 * @var array
+	 */
+	public static $email_classes = [
```

Comment:
I move the email classes into a list like this so that I could easily create a function to check if the email being previewed is a subscription email (see `is_subscription_email()`)